src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_alias_commands.py

```diff
@@ -0,0 +1,166 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Minor: It might be a better idea to split this into two changes, since they are unrelated.  One to fix the update methods and another change to split the test into individual files.